packages/manager/apps/hycu/src/pages/dashboard/Dashboard.page.tsx

```diff
@@ -36,29 +37,57 @@ export type DashboardLayoutProps = {
   tabs: DashboardTabItemProps[];
 };
 
+const LicenseSuspendedBanner = () => {
+  const { t } = useTranslation('hycu/dashboard');
+
+  const notification = {
+    content: t('hycu_dashboard_warning_license_suspended_message'),
+    dismissable: false,
+    type: NotificationType.Warning,
+    uid: 999,
+  };
+
+  return <OdsNotification notification={notification} />;
```

Comment:
why not use `<OsdsMessage />` since it's a simple message banner and not a notification?

Reply:
To have exactly the same behavior and style